docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -226,3 +229,29 @@ and a file system location of ``/var/my_tables/test_table``::
         format = 'PARQUET',
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
+
+.. _iceberg-materialized-views:
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+The Iceberg connector supports :ref:`sql-materialized-views-management`. In the
+underlying system each materialized view consists of a view definition and an
+Iceberg storage table. The storage table name is stored as a materialized view
+property. The data is stored in that storage table.
+
+You can use the :ref:`iceberg-table-properties` to control the created storage
+table and therefore the layout and performance. For example, you can use the
```

Comment:
Do we have a list of table properties that are supported for a materialized view?

Reply:
The section it links to has that..